docker/config/default.config.envsubst

```diff
@@ -9,6 +9,9 @@ network.host_ip=$RADIXDLT_HOST_IP_ADDRESS
 network.p2p.seed_nodes=$RADIXDLT_NETWORK_SEEDS_REMOTE
 network.p2p.use_proxy_protocol=$RADIXDLT_NETWORK_USE_PROXY_PROTOCOL
 
+state_hash_tree.gc.interval_sec=$RADIXDLT_STATE_TREE_GC_INTERVAL_SEC
```

Comment:
Should this be `state_tree` now on the LHS?

Reply:
Yes - there will be a PR which pulls the new scrypto dep (with the rename there) and applies the rename to existing symbols in Node, including this one.